packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1101,6 +1102,52 @@ abstract class InheritedWidget extends ProxyWidget {
   bool updateShouldNotify(InheritedWidget oldWidget);
 }
 
+/// Defines a location in the widget tree that an [InheritedWidgetLinkChild] can
```

Comment:
Ideally these classes would be outside the framework.dart file. Is that possible?